src/actions/receive-profile.js

```diff
@@ -634,7 +634,7 @@ export function errorReceivingProfileFromFile(error: Error): Action {
   };
 }
 
-function _fileReader(input) {
+function _fileReader(input: File) {
```

Comment:
nit: please type the return result, using `*` to let Flow infer from the content. I _think_ Flow sees it differently than when we put nothing.

Reply:
What I noticed in the past is that Flow doesn't resolve the type at the same moment in its process. There has been differences depending whether we have this. I wouldn't normally care but because you're using `typeof _fileReader` below I'd rather be safer.

In my mind, the difference is:
* using the star: "I want the return type to be defined, but I want you to infer it from what you can statically infer from this function source code". So at the phase of "finding out all specified types" the function is fully typed, and "typeof" will give the correct type.
* not using it: Flow will try to match how the function is called and how

Reply:
What I noticed in the past is that Flow doesn't resolve the type at the same moment in its process. There has been differences depending whether we have this (this happened around the mapStateToProps function before we used our explicit way). I wouldn't normally care but because you're using `typeof _fileReader` below I'd rather be safer.

In my mind, the difference is:
* using the star: "I want the return type to be defined, but I want you to infer it from what you can statically infer from this function source code". So at the phase of "finding out all specified types" the function is fully typed, and "typeof" will give the correct type.
* not using it: Flow will try to match how the function is called and what it does. This means it may not be fully typed at the moment `typeof` is resolved.

I'm not sure at all this is the correct way of thinking about it. But this is what I understood from previous issues.